src/Components/RideInfo/RideInfo.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { StyleSheet, Text, View, ViewStyle } from "react-native";
+import RideProperty from "./Components";
+import fonts from "src/Themes/Fonts";
+import withTheme, { ThemeProps, ThemeParamsType } from "src/Themes/withTheme";
+
+type Style = ReturnType<typeof getStyles>;
+
+interface Props extends ThemeProps<Style> {
+  properties: { label: React.ReactNode; content: React.ReactNode }[];
+  style?: ViewStyle;
+}
+
+export class PureRideInfo extends React.Component<Props> {
+  public render() {
+    const { properties, styles } = this.props;
+    return (
+      <View style={[styles.container, this.props.style]}>
+        <View style={styles.propertiesRawContainer}>
+          {properties.map((property, index) => (
+            <RideProperty
+              key={typeof property.label === "string" ? property.label : index.toString()}
```

Comment:
You can not use React.ReactNode as key here.

If any component will be passed here, the components' `toString()` will be `[Object Object]`. So all the properties will have an equal key and only one item will be rendered.

Reply:
I don't think so. `typeof property.label === "string" ? property.label : index.toString()`

Reply:
Ok, my mistake, didn't see it is about an index.

BUT

[Using index as key is an anit-pattern](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)

Reply:
I know this, and this is why I use `property.label` when I can, but sometimes I can't so I use index.